examples/js/postprocessing/AdaptiveToneMappingPass.js

```diff
@@ -186,7 +186,7 @@ THREE.AdaptiveToneMappingPass.prototype = {
 			this.previousLuminanceRT.dispose();
 
 		}
-		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBFormat };
+		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBAFormat };
```

Comment:
@WestLangley . I just ran into this today in my code and needed to change my render buffer to RGBA. Apparently, in the ```WEBGL_color_buffer_float``` extension, only RGBA32F is renderable. See https://www.khronos.org/registry/webgl/extensions/WEBGL_color_buffer_float/

In OpenGL ES, there's more detail of what's supported for renderbuffers and what is only for textures: https://www.khronos.org/registry/gles/extensions/EXT/EXT_color_buffer_float.txt. Note that RGB32F is a texture-only format.

Interestingly, if you use half-float buffers, you can render to a target without alpha. See https://www.khronos.org/registry/webgl/extensions/EXT_color_buffer_half_float/